### PR TITLE
dev: Update the webpack config to alias crypto

### PR DESCRIPTION
**Pull request by nturinski**
Also removes clean-webpack because all it has done it cause me pain

### CODE REVIEW

File: dev/src/webpack/getDefaultWebpackConfig.ts [hunk 1]
```diff
@@ -254,17 +262,6 @@ export function getDefaultWebpackConfig(options: DefaultWebpackOptions): webpack
         ]
     };
 
-    // Clean the dist folder before webpacking
-    if (!options.suppressCleanDistFolder) {
```
Comment (alexweininger) [inline dev/src/webpack/getDefaultWebpackConfig.ts:258]: There's an option to suppress it, should we just use that?

**Review by alexweininger — COMMENTED** [review]
Did you mean to include some utils changes in this PR?

**Comment by nturinski** [comment]
Crap. I did not.

File: dev/src/webpack/getDefaultWebpackConfig.ts (hunk 1 above)
Comment (nturinski) [reply dev/src/webpack/getDefaultWebpackConfig.ts:258]: Uggggh I guess we could. It was nice to delete something that hurt me though.

**Comment by nturinski** [comment]
Closing in favor of https://github.com/microsoft/vscode-azuretools/compare/dev/nat/webpackConfig?expand=1

**Pull request closed without merging**